Graffino/Browser-Update v2.0.2 not found

Trying to `make clean && make` today gives:
```
cd build && /home/phil/.yarn/bin/yarn install
yarn install v1.22.0
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error An unexpected error occurred: "https://codeload.github.com/Graffino/Browser-Update/tar.gz/0edda8c7f53ad2cbe68538c00b8c790513096d60: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/home/phil/git/owncloud/core/build/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Makefile:166: recipe for target 'build/node_modules' failed
make: *** [build/node_modules] Error 1
```

The similar problem happened in a nightly build https://drone.owncloud.com/owncloud/wopi/813/2/2
```
error An unexpected error occurred: "https://codeload.github.com/Graffino/Browser-Update/tar.gz/0edda8c7f53ad2cbe68538c00b8c790513096d60: Request failed \"404 Not Found\""
```

It seems that https://github.com/Graffino/browser-update/releases has new releases like `v2.0.3` but the old tags are gone - no more `v2.0.2`

Probably we will have to update to `v2.0.3`

--- a/build/package.json
+++ b/build/package.json
@@ -15,7 +15,7 @@
     "@bower_components/blueimp-md5": "blueimp/JavaScript-MD5#1.0.1",
     "@bower_components/bootstrap": "twbs/bootstrap#3.4.1",
     "@bower_components/bowser": "ded/bowser#1.9.4",
-    "@bower_components/browser-update": "Graffino/Browser-Update#v2.0.2",
+    "@bower_components/browser-update": "Graffino/Browser-Update#v2.0.5",
     "@bower_components/clipboard": "zenorocha/clipboard.js#v2.0.4",
     "@bower_components/davclient.js": "owncloud/davclient.js#0.1.3",
     "@bower_components/es6-promise": "components/es6-promise#4.2.4",

--- a/changelog/unreleased/36976
+++ b/changelog/unreleased/36976
@@ -1,0 +1,4 @@
+Change: Update Graffino/Browser-Update from 2.0.2 to 2.0.5
+
+https://github.com/owncloud/core/issues/36976
+https://github.com/owncloud/core/pull/36981

